src/Renci.SshNet/Messages/Message.cs

```diff
@@ -55,58 +56,61 @@ internal byte[] GetPacket(byte paddingMultiplier, Compressor compressor)
 
             var messageLength = BufferCapacity;
 
-            SshDataStream sshDataStream;
-
-            if (messageLength == -1 || compressor != null)
+            if (messageLength == -1 || compressor is not null)
             {
-                sshDataStream = new SshDataStream(DefaultCapacity);
+                using (var sshDataStream = new SshDataStream(DefaultCapacity))
```

Comment:
An `SshDataStream` is a `MemoryStream` which doesn't do anything on disposal. The change is fine but unnecessary.